extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -135,6 +135,14 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		}
 	}
 
+	function updateTargetingForBlockedTraffic() {
+		win.googletag.pubads().getSlots().forEach(function(slot) {
+			// slot.clearTargeting described in docs is not applicable in this context
+			slot.targeting.src = [];
```

Comment:
slots are defined with `src = ['gpt']` and `slot.setTargeting('src', 'rec');` will make it `src = ['gpt', 'rec']`